cmd/install/assets/assets.go

```diff
@@ -34,7 +34,13 @@ var capiResources = map[string]string{
 	"cluster-api-provider-aws/infrastructure.cluster.x-k8s.io_awsclusters.yaml":                      "v1beta1",
 	"cluster-api-provider-aws/infrastructure.cluster.x-k8s.io_awsmachines.yaml":                      "v1beta1",
 	"cluster-api-provider-aws/infrastructure.cluster.x-k8s.io_awsmachinetemplates.yaml":              "v1beta1",
-	"cluster-api-provider-ibmcloud/infrastructure.cluster.x-k8s.io_ibmvpcclusters.yaml":              "v1alpha4",
+	"cluster-api-provider-ibmcloud/infrastructure.cluster.x-k8s.io_ibmpowervsclusters.yaml":          "v1beta1",
+	"cluster-api-provider-ibmcloud/infrastructure.cluster.x-k8s.io_ibmpowervsimages.yaml":            "v1beta1",
+	"cluster-api-provider-ibmcloud/infrastructure.cluster.x-k8s.io_ibmpowervsmachines.yaml":          "v1beta1",
+	"cluster-api-provider-ibmcloud/infrastructure.cluster.x-k8s.io_ibmpowervsmachinetemplates.yaml":  "v1beta1",
+	"cluster-api-provider-ibmcloud/infrastructure.cluster.x-k8s.io_ibmvpcclusters.yaml":              "v1beta1",
+	"cluster-api-provider-ibmcloud/infrastructure.cluster.x-k8s.io_ibmvpcmachines.yaml":              "v1beta1",
```

Comment:
good point, I thought will tag all the available crds so that they are tagged with right version.

Reply:
don't mind un-doing which is not really affecting powervs thing, lemme revert back the changed.

Reply:
mm not sure how you mean by tagging, but the PR is adding CRDs which are not used. We should only install what is needed.

Reply:
>well, this block of code just adds additional labels to the crds

The PR before was introducing a new CRD `ibmvpcmachines.yaml` which is not nor used nor needed, we don't want to do that.

AWS contains what the aws capi controller requires to work and if it we are adding something else unnecessarily, that's a bug on us. That means we need to fix the bug, not introducing more bugs in other platforms :)

Reply:
I believe all those crds are required anyways because controller code does contain the `Reconciler` block for all those crds, it will fail if any of those crds missing in the cluster.